### PR TITLE
Discovery filter

### PR DESCRIPTION
**Pull request by secretrobotron**
~~## WARNING: NEEDS MIGRATION TO MERGE COLUMNS FOR EXISTING USERS~~

Contributes to fixing https://github.com/mozilla/api.webmaker.org/issues/121

### CODE REVIEW

**Comment by secretrobotron** [comment]
@cadecairos r?

File: scripts/create-tables.sql [hunk 1]
```diff
@@ -6,8 +6,7 @@ CREATE TABLE IF NOT EXISTS "users"
   created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   deleted_at timestamp DEFAULT NULL,
-  language varchar NOT NULL DEFAULT 'en',
-  country varchar NOT NULL DEFAULT 'US',
+  language varchar NOT NULL DEFAULT 'en-US',
```
Comment (cadecairos) [inline scripts/create-tables.sql]: we're going to make the modifications in the migration script, so don't change this here

Comment (cadecairos) [inline scripts/create-tables.sql]: actuially nvm, this is cool.

**Comment by cadecairos** [comment]
R+ from me

**Pull request merged**